src/Item_Project.php

```diff
@@ -235,8 +239,9 @@ public function getTabNameForItem(CommonGLPI $item, $withtemplate = 0)
                 default:
                    // Not used now
                     if (
-                        Session::haveRight("project", Project::READALL)
-                        && ($item instanceof CommonDBTM)
+                        Session::haveRight("project", Project::READMY)
+                        && $item instanceof CommonDBTM
+                        && in_array($item->getType(), $CFG_GLPI["project_asset_types"])
                     ) {
                         if ($_SESSION['glpishow_count_on_tabs']) {
                               // Direct one
```

Comment:
Can you check if this counter is correct? I am not sure we should count `Linked items`, as they are not shown in the tab itself.